examples/buildkit0/buildkit.go

```diff
@@ -1,3 +1,4 @@
+// +build ignore
```

Comment:
what's this?

Reply:
Removed. (I had another test script in the same dir that caused the build as pkg to fail)